tests/test_conditions.py

```diff
@@ -42,6 +47,43 @@ def test_interval_condition():
     )
 
 
+def test_normalization_interval_condition():
+    condition = IntervalCondition(p=0.5, min=10, max=100)
+    assert condition.get_normalized(10, 1000).get_denormalized(10, 1000) == condition
```

Comment:
Assign all 10s and 1000s in these functions to variables, then loop over a few different values for each?